lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -71,7 +70,7 @@ func ImageCache() *ICache {
 
 // Update runs only once at startup to hydrate the image cache
 func (ic *ICache) Update(client *client.PortLayer) error {
-	log.Debugf("Updating image cache...")
+	log.Debugf("Updating image cache")
```

Comment:
There's actually an `Infof` before the call to this guy over in the persona init.